test/geocel/data/lar-sphere.gdml

```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<gdml xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://service-spi.web.cern.ch/service-spi/app/releases/GDML/schema/gdml.xsd">
+
+  <define>
+    <matrix name="RI" coldim="2" values="1.8785e-06 1.2221243542166 1.8940e-06 1.2222029377323 1.9101e-06 1.2222825413456 1.9261e-06 1.2223604051795 1.9424e-06 1.2224381070094 1.9590e-06 1.2225157688711 1.9762e-06 1.2225949497339 1.9933e-06 1.2226728864441 2.0108e-06 1.2227511290774 2.0289e-06 1.2228312481231 2.0470e-06 1.2229104393633 2.0654e-06 1.2229902616388 2.0845e-06 1.2230723199947 2.1036e-06 1.2231537374513 2.1230e-06 1.2232361043232 2.1428e-06 1.2233195267102 2.1634e-06 1.2234056897612 2.1840e-06 1.2234915704979 2.2049e-06 1.2235788347965 2.2267e-06 1.2236692538263 2.2485e-06 1.2237596566990 2.2708e-06 1.2238517915638 2.2939e-06 1.2239475391085 2.3170e-06 1.2240435454107 2.3406e-06 1.2241416640706 2.3648e-06 1.2242420302676 2.3898e-06 1.2243467108633 2.4150e-06 1.2244520480945 2.4406e-06 1.2245600757483 2.4673e-06 1.2246730352174 2.4942e-06 1.2247869894369 2.5215e-06 1.2249041417343 2.5501e-06 1.2250269429781 2.5787e-06 1.2251511263208 2.6080e-06 1.2252790990981 2.6380e-06 1.2254110795713 2.6692e-06 1.2255498628245 2.7006e-06 1.2256906509598 2.7327e-06 1.2258361874525 2.7663e-06 1.2259895862066 2.8000e-06 1.2261455681055 2.8346e-06 1.2263071887802 2.8707e-06 1.2264779455048 2.9070e-06 1.2266519928105 2.9443e-06 1.2268327625990 2.9825e-06 1.2270206659616 3.0225e-06 1.2272198383341 3.0629e-06 1.2274235186718 3.1043e-06 1.2276357672922 3.1476e-06 1.2278613234401 3.1914e-06 1.2280925883611 3.2363e-06 1.2283342192762 3.2835e-06 1.2285917000180 3.3311e-06 1.2288564336687 3.3802e-06 1.2291338166016 3.4307e-06 1.2294247402393 3.4837e-06 1.2297359753784 3.5374e-06 1.2300572815744 3.5927e-06 1.2303953408143 3.6509e-06 1.2307581975201 3.7099e-06 1.2311340756410 3.7708e-06 1.2315309395240 3.8350e-06 1.2319584795948 3.9001e-06 1.2324030531751 3.9675e-06 1.2328742967333 4.0373e-06 1.2333745182601 4.1109e-06 1.2339164497558 4.1858e-06 1.2344833000133 4.2636e-06 1.2350878388706 4.3457e-06 1.2357460538231 4.4296e-06 1.2364381478993 4.5167e-06 1.2371803058492 4.6091e-06 1.2379930862779 4.7035e-06 1.2388529939487 4.8019e-06 1.2397811112689 4.9044e-06 1.2407852902585 5.0135e-06 1.2418956339948 5.1254e-06 1.2430824604497 5.2425e-06 1.2443774409398 5.3673e-06 1.2458225245477 5.4958e-06 1.2473825079216 5.6305e-06 1.2491028277676 5.7748e-06 1.2510450704786 5.9238e-06 1.2531685960320 6.0806e-06 1.2555430061400 6.2461e-06 1.2582127677221 6.4241e-06 1.2612929591155 6.6090e-06 1.2647427641559 6.8048e-06 1.2687046921432 7.0166e-06 1.2733876575744 7.2378e-06 1.2787793729478 7.4734e-06 1.2851700143090 7.7297e-06 1.2930098842738 7.9990e-06 1.3024417515349 8.2877e-06 1.3142220339702 8.5981e-06 1.3293224516058 8.9390e-06 1.3497665943227 9.3011e-06 1.3776746680090 9.6938e-06 1.4188784984198 1.0129e-05 1.4873838729402 1.0597e-05 1.6167515615703"/>
```

Comment:
Where does this data come from? (looks very reasonable values, but just curious about the reference) - probably may add the reference in the test.

Reply:
These I actually got from [this refractive index database](https://refractiveindex.info/?shelf=main&book=Ar&page=Grace-liquid-90K), with these particular values calculated using this reference:

E. Grace, A. Butcher, J. Monroe, J. A. Nikkel. Index of refraction, Rayleigh scattering length, and Sellmeier coefficients in solid and liquid argon and xenon, [Nucl. Instr. Meth. Phys. Res. A 867, 204-208 (2017)](https://doi.org/10.1016/j.nima.2017.06.031)

I'll make a note in the test.

Reply:
Great!  There are even Sellmeier coefficients from the link which can be used for a functional input later.